tests/foreman/cli/test_activationkey.py

```diff
@@ -816,21 +815,18 @@ def test_positive_delete_manifest(function_org):
 
     :CaseAutomation: Automated
     """
-    with manifests.clone() as manifest:
-        upload_manifest(function_org.id, manifest.content)
-    new_ak = make_activation_key({'organization-id': function_org.id})
-    ak_subs = ActivationKey.subscriptions({'id': new_ak['id'], 'organization-id': function_org.id})
+    org = function_entitlement_manifest_org
+    new_ak = make_activation_key({'organization-id': org.id})
+    ak_subs = ActivationKey.subscriptions({'id': new_ak['id'], 'organization-id': org.id})
```

Comment:
```suggestion
    ak_subs = target_sat.cli.ActivationKey.subscriptions({'id': new_ak['id'], 'organization-id': org.id})
```

Reply:
Same everywhere ?

Reply:
This change can be done in separate PR at it is not in scope for this PR. We can do this change in another PR.

Reply:
Cool!